target/linux/ar71xx/base-files/etc/board.d/01_leds

```diff
@@ -181,6 +181,9 @@ cf-e320n-v2)
 	ucidef_set_led_netdev "wan" "WAN" "$board:red:wan" "eth1"
 	ucidef_set_led_wlan "wlan" "WLAN" "$board:blue:wlan" "phy0tpt"
 	;;
+cf-e355ac)
+	ucidef_set_led_netdev "lan" "LAN" "$board:green:lan" "eth0"
```

Comment:
Don't duplicate code and move this line down, please (include it with other COMFAST boards).